lambdas/url_access_patterns/lambda_for_generateURL/index.js

```diff
@@ -0,0 +1,48 @@
+const AWS = require("aws-sdk");
+const DT = require("date-and-time");
+const crypto = require("crypto");
+const dynamo = new AWS.DynamoDB.DocumentClient();
+const Hashids = require("hashids");
+
+function generateUniqueId() {
+  const salt = crypto.randomBytes(16).toString("base64");
+  const uniqueId = new Hashids(salt, 6);
+  const id = uniqueId.encode(Math.ceil(Math.random() * 1000));
+  return id;
+}
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: message,
+  };
+}
+
+function generateTimestamp() {
+  const ts = Date.now();
+  const pattern = DT.compile("YYYY-MM-DD-HH-mm-ss");
+  const timestamp = DT.format(new Date(ts), pattern);
+}
+
+exports.generateURL = async (event) => {
+  const urlID = generateUniqueId();
+  const reqBody = event.body;
+  const timestamp = generateTimestamp();
+  try {
+    await dynamo
+      .put({
+        TableName: "V-Transfer",
+        Item: {
+          PK: `FILE#${reqBody.f_timestamp}`,
+          SK: `URL#${timestamp}`,
+          GS1_PK: `${urlID}`,
+          hash: `${reqBody.hash}`,
+          visible: `${reqBody.visible}`,
+          clicks_left: reqBody.clicks_left ? reqBody.clicks_left : 50,
+        },
+      })
+      .promise();
+    return response(200, urlID);
+  } catch (err) {
+    return response(200, err.message);
```

Comment:
change the status code, in case of error

Reply:
We have to give a static status code and it is to be done for all lambdas so we thought of opening a new issue for lambdas. So he has left it as it is.

Reply:
then why @lakshya-20  haven't you approved the PR?

Reply:
This must be addressed with reference to the issue #39